mono/metadata/Makefile.am

```diff
@@ -180,6 +180,7 @@ common_sources = \
 	image-internals.h	\
 	jit-info.c		\
 	loader.c		\
+	loader-internals.h	\
```

Comment:
I wonder if we need more levels here, like for functions and opaque types, vs. revealed types and globals. This isn't particularly internal -- it isn't loader internal, just runtime internal.

Reply:
We definitely don't need more level 😉